### PR TITLE
Arabic Translation

### PR DESCRIPTION
**Pull request by AviationGamerX**
This is my first translation, in OpenTTD in general. I like your works, keep the hard work going. I am not sure if this might come out perfect as a translation, if there is anything wrong I will probably send you an edit. Thank you!

### CODE REVIEW

File: src/lang/Arabic
Comment (2TallTyler) [inline src/lang/Arabic:1]: The language ID for Arabic (Egypt) is 0x14.

**Review by 2TallTyler — CHANGES_REQUESTED** [review]
Thanks very much for your contribution!

Two things to fix:
- The file name should be `arabic.lng`
- The language ID needs to be changed from 0x01 to 0x14 (see my comment on the code to show you where)

If if helps you check translations in-game, I made these quick fixes and built the NewGRF: [improved_town_industries.zip](https://github.com/2TallTyler/improved_town_industries/files/5720295/improved_town_industries.zip)

I can't push my fixes directly to your repo, since I believe you unchecked 'Allow edits and access to secrets by maintainers', so you will still need to make them and amend the PR. If you can squash your commits into one, that would be great, otherwise I can squash after merging.

Thanks!

**Comment by AviationGamerX** [comment]
So I am not sure why you can't edit, it allow edits by maintainers box is checked. Anyway, I have made the changes listed above, I tried out the zip file, the only problem is the brackets and punctuation is out of place, other than that everything else seems to be working, I can't figure out a way to solve the punctuation thing, otherwise, it's good to go. If the permission is an issue, u can just copy and paste it as a new file. Anyways, thanks for your co-operation, Thanks!

**Review by 2TallTyler — APPROVED** [review]
Ready to merge whenever you confirm you're happy with the translations using the .grf I built.

**Comment by AviationGamerX** [comment]
> Ready to merge whenever you confirm you're happy with the translations using the .grf I built.

good to go

**Pull request merged**
